app/views/tutorials/overview.html.erb

```diff
@@ -1,113 +1,137 @@
+              <%= link_to a.title,
+                          lecture_tutorial_overview_path(@lecture,
+                                                        params: { assignment: a.id }),
+                          class: 'dropdown-item' %>
+            <% end %>
+          </div>
+        </span>
+      <% else %>
+        <button class="btn btn-secondary" disabled>
           <%= @assignment.title %>
         </button>
-        <div class="dropdown-menu">
-          <% (@assignments - [@assignment]).each do |a| %>
-            <%= link_to a.title,
-                        lecture_tutorial_overview_path(@lecture,
-                                                       params: { assignment: a.id }),
-                        class: 'dropdown-item' %>
+      <% end %>
+
+      <% if @tutorials.size > 0 %>
+        <span class="dropdown ml-3">
+          <button class="btn btn-secondary dropdown-toggle"
+                  data-toggle="dropdown"
+                  type="button">
+            <%= t('basics.overview') %>
+          </button>
+          <div class="dropdown-menu">
+            <% (@tutorials - [@tutorial]).each do |t| %>
+              <%= link_to t.title,
+                          lecture_tutorials_teacher_path(@lecture,
+                                                        params: { assignment: @assignment.id,
+                                                                  tutorial: t.id }),
+                          class: 'dropdown-item' %>
+            <% end %>
+          </div>
+        </span>
+      <% else %>
+        <button class="btn btn-secondary" disabled>
+          <%= t('basics.tutorial') %>
+        </button>
+      <% end %>
+
+      <% if current_user.in?(@lecture.tutors) %>
+        <%= link_to t('tutorial.own_tutorials'),
+                    lecture_tutorials_path(@lecture),
+                    class: 'btn btn-primary ml-3' %>
+      <% end %>
+      <table class="table table-bordered mt-3">
+        <thead class="thead-light">
+          <tr>
+            <th sope="col">
+              <%= t('basics.tutorial') %>
+            </th>
+            <th scope="col">
+              <%= t('basics.tutors') %>
+            </th>
+            <th scope="col">
+              <%= t('basics.submissions') %>
+            </th>
+            <th scope="col">
+              <%= t('basics.corrections') %>
+            </th>
+            <th scope="col">
+              <%= t('submission.late_submissions') %>
+            </th>
+          </tr>
+        </thead>
+        <tbody style="max-height: 50px; overflow-y: scroll;">
+          <% @lecture.tutorials.natural_sort_by(&:title).each do |t| %>
+            <tr>
+              <td>
+                <%= t.title %>
+              </td>
+              <td>
+                <%= t.tutor_names || t('basics.tba') %>
+              </td>
+              <td>
+                <%= Submission.where(tutorial: t, assignment: @assignment)
```

Comment:
This was not introduced by you, so I am sorry to bother you with it, but I think we should use the chance to change it. The code for this and the following queries should not be within the view. There should be methods in some model that do these queries. (Additionally, as these queries happen to occur in a loop, I guess we have some n+1 action going on here (which should not really be problematic as the number of tutorials i alecture should be rather small, but still)).